### PR TITLE
added DRF logging

### PR DESCRIPTION
**Pull request by eweisner1973**

### CODE REVIEW

File: apps/user/views.py [hunk 1]
```diff
@@ -31,11 +31,32 @@ class UserCreate(generics.CreateAPIView):
...
+    # logging_methods = ['GET','POST']
     queryset = get_user_model().objects.all()
     serializer_class = UserSerializer
     permission_classes = (permissions.AllowAny,)
 
+    '''
+    optional mask sensitive data. By default drf-tracking is hiding the
+    values of the fields {'api', 'token', 'key', 'secret', 'password', 'signature'}.
+    '''
+    sensitive_fields = {'password'}
```
Comment (weslord) [inline apps/user/views.py]: Does setting this to just password override the other defaults (ie no longer hides 'api', 'token', 'key', 'secret', 'signature')?

**Comment by weslord** [comment]
Is there a reason for calling the app 'support' rather than something like 'logging' or 'logs'?

File: apps/user/views.py (hunk 1 above)
Comment (eweisner1973) [reply apps/user/views.py]: I didn't test that specifically, but I wouldn't think so.

**Comment by eweisner1973** [comment]
> Is there a reason for calling the app 'support' rather than something like 'logging' or 'logs'?

Nope. I figured we might add other diagnostic tool to the app, so just wanted something generic. I am not married to the name by any means.

**Pull request merged**